include/f_core/flight/c_phase_controller.h

```diff
@@ -6,8 +6,9 @@
 #include <zephyr/kernel.h>
 
 int flight_log_init(const char *flight_log_file_name, fs_file_t *fp);
-int flight_log_source_event(bool enabled, const char *source, const char *event);
-int flight_log_event_confirmed(bool enabled, const char *event, bool currentState);
+int flight_log_source_event(bool enabled, fs_file_t *fp, const char *source, const char *event);
```

Comment:
Rationale for not making these part of the class?

Reply:
I wanted to include a LOG_INF alongside the log to a file And zephyr doesn't want you to log midle define in a header cuz it could screw up other things. so by having them separate they're not part of the templTe and can be normal in the cpp file